app/src/main/res/values/styles_parents.xml

```diff
@@ -64,10 +64,11 @@
     </style>
 
     <style name="Theme.VinylMusicPlayer.Base.Black" parent="@style/Theme.VinylMusicPlayer.Base">
-        <item name="android:windowBackground">@android:color/black</item>
+        <item name="android:colorBackground">@android:color/black</item>
+		<item name="android:windowBackground">@android:color/black</item>
```

Comment:
Can you please fix the indent here?

Reply:
Should be fixed now as requested, apologies for the delay.